src/PeopleSDKAdapter.js

```diff
@@ -53,6 +53,17 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
     };
   }
 
+  /**
+   * Returns an observable that emits me data of.
+   *
+   * @public
+   * @returns {Observable.<Person>}
+   * @memberof PeopleSDKAdapter
+   */
+  getMe() {
+    return from(this.datasource.people.get('me')).pipe(flatMap(({id}) => this.getPerson(id)));
```

Comment:
Why bother calling `people.get` at all? Can't you just return `this.getPerson('me')`?

Reply:
to listen to all status changes. and return the mapped person Model.

Reply:
just re-read your comment. we can't do that, because `deconstructHydraId` method invoked inline 78 inside `getPerson` method definition from the common library expects an actual ID